innovation_sweet_spots/getters/inputs.py

```diff
@@ -0,0 +1,48 @@
+"""
+Modules for collecting data analysis inputs.
+
+To fetch all inputs, simply run "python inputs.py"
+"""
+from innovation_sweet_spots import PROJECT_DIR, config, logging
+from pathlib import Path
+from data_getters.gtr import build_projects
+import pandas as pd
+
+gtr_path = f"{PROJECT_DIR}/inputs/data/gtr_projects.csv"
+
+
+def get_gtr_projects(fpath=gtr_path, fields=["id"]):
+    """
+    Downloads GTR projects from Nesta database and stores them locally.
+    Function can be used from command line as follows:
+        python -c "from innovation_sweet_spots.getters.inputs import get_gtr_projects; get_gtr_projects();"
+
+    Parameters
+    ----------
+    fpath : str
+        Location on disk for saving the projects table
+    fields : list of str
+        Use the default value; for additional functionality see 'data_getters' documentation
+
+    Returns
+    -------
+    df : pandas.DataFrame
+        Table with projects
+    """
+    logging.info(f"Collection of GTR projects in progress")
+    projects = build_projects(
+        config_path=config["database_config_path"],
+        chunksize=5000,
+        table_wildcards=["gtr_projects"],
+        desired_fields=fields,
+    )
+    df = pd.DataFrame(projects)
+    df.to_csv(fpath, index=False)
+    n_projects = len(df)
+    logging.info(f"Collected {n_projects} GTR projects and stored them in {fpath}")
```

Comment:
Suggest not to re-download if a flag `use_cached=False` or the file doesn't already exist. I'd recommend that `use_cached=True`  by default, but that's a design decision for you

```suggestion
    if not (use_cached or os.path.exists(fpath)):
        projects = build_projects(
            config_path=config["database_config_path"],
            chunksize=5000,
            table_wildcards=["gtr_projects"],
            desired_fields=fields,
        )
        with open(fpath, "w") as f:
            json.dump(projects, f)
        logging.info(f"Collected {len(projects)} GTR projects and stored them in {fpath}")
    df = pd.read_json(fpath, orient='records')
    return df
```

Reply:
Sorry @jaklinger, I'm a bit confused about the `use_cached` flag. Should it be like another input variable to the get_gtr_projects() ? 

If `use_cache=True`, should it check the "cache" somewhere, or it mean to simply say not to download anything?

Reply:
Oh, do I need to use the `@lru_cache` here?

Reply:
exactly, so by cache i just mean that you already have a local copy (i.e. `os.path.exists(fpath)`) and so add `use_cached=True` to the kwargs for `get_gtr_projects` to avoid re-downloading the data every time